recipe/meta.yaml

```diff
@@ -36,16 +37,14 @@ about:
   license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE.txt
-  summary: 'This is a small library to manage continuous streams of data,
-  particularly when complex branching and control flow situations arise.'
+  summary: "Manage streaming data, optionally with Dask and Pandas"
 
   description: |
-    This is a small library to manage continuous streams of data, particularly when complex branching and control flow situations arise.
+    Manage streaming data, optionally with Dask and Pandas
   doc_url: https://streamz.readthedocs.io/en/latest/
   dev_url: https://github.com/mrocklin/streamz/
 
 extra:
   recipe-maintainers:
     - CJ-Wright
     - mrocklin
-    - ordirules
```

Comment:
I think he goes by @jrmlhermitte now

Reply:
oh yes sorry about change and thanks @CJ-Wright